how-to-verify-keys.md

```diff
@@ -0,0 +1,34 @@
+Getting cela's Public Key
+========================
+
+1. Get an _untrusted_ copy of cela's key signature from the Repository Security section in homeworld's `README.md`.
+2. Get an _untrusted_ copy of cela's key from the MIT keyserver: `gpg --keyserver pgp.mit.edu --recv-keys <last 8 digits of untrusted cela's key signature>`
+3. Compute the fingerprint of the received key: `gpg --fingerprint cela`
+4. Ask cela for her fingerprint and check against the output of the previous command.
+5. Set the trust on cela's key with `gpg --edit-key cela`, `trust`, `4`, `quit` (`5` is reserved for your own key).
+
+Getting a Trusted Copy of `trust.txt`
+=====================================
+
+1. Clone the homeworld repository.
+2. Get the signatures of the commits for `trust.txt`: `git log --show-signature trust.txt`
+3. Check that the latest commit in the log is "good".
```

Comment:
Perhaps you want to have different criteria: not that the latest commit is "good", but checking who it says it's signed by? It seems like you might end up having previously trusted some other person's key, and then if they tried to sign these commits, it would also show up as "good", even though they aren't supposed to be signing these commits.